sycl/include/CL/sycl/usm.hpp

```diff
@@ -12,6 +12,7 @@
 #include <CL/sycl/usm/usm_enums.hpp>
 
 #include <cstddef>
+#include <tuple>
```

Comment:
Could you please add more info about what and why this patch does in the commit message? To me it's not obvious what get_pointer_info is.

Reply:
OK.